src/modules/flow/persistence/persistence.json

```diff
@@ -0,0 +1,259 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "fs",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD 3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "io/persistence",
+      "description": "Persists the last input packet's data it receives to storage, also repeating that packet on output port. Note that upon a new connection on the unique output port, if a previous stored packet is found, it will already be sent.",
+      "methods": {
+        "close": "persist_close",
+        "open": "persist_boolean_open"
+      },
+      "name": "persistence/boolean",
+      "options": {
+        "members": [
+          {
+            "data_type": "string",
+            "description": "Name to persist a boolean packet. ",
+            "name": "name"
+          },
+          {
+            "data_type": "string",
+            "description": "Storage where data will be persisted. It can be of the following: fs, efivars",
+            "name": "storage"
+          },
+          {
+            "data_type": "string",
+            "default": null,
+            "description": "If 'storage' is 'fs', directory path where data will be persisted. If 'storage' different of 'fs', this option will be ignored.",
+            "name": "fs_dir_path"
```

Comment:
you can remove this one to keep the API the same. What you do is take the path in the `name` option above. If `efivars` and `name` is not compatible, then you will try to open the file and it will fail anyway.